samples/Sentry.Samples.AspNetCore.Mvc/Sentry.Samples.AspNetCore.Mvc.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.2"  />
```

Comment:
@bruno-garcia btw the guidance is still to remove the `Version=` reference and let the sdk keep it at 2.1.1 for framework-dependent builds and change it for self-contained ones automatically. Libraries should target the lowest version of abstraction libraries. In reality, they should target 2.1.0 or 2.1.1 but not higher versions, as they would override the implicit M.ANC.App/2.1.1 reference that's used during build and produce a framework-dependent app that runs on say a 2.1.5 sharedfx and then have some 2.1.2 DLLs deployed alongside them just because they were built using the 2.1.1 package.

Reply:
@dasMulli that makes sense! Thanks a lot for pointing it out.
I'll make sure next version goes as advised.

Reply:
it's just a sample, so it doesn't hurt.
The libraries should be fine using the 2.1.1 versions for now AFAIK but I don't suggest updating them any further.